fieldtracing/fieldtracing.cpp

```diff
@@ -793,34 +790,40 @@ namespace FieldTracing {
       creal stepSize = min(1000e3, technicalGrid.DX / 2.);
       std::vector<Real> cellFWTracingStepSize(globalDccrgSize, stepSize); // In-flight storage of step size, needed when crossing into next MPI domain
       std::vector<Real> cellBWTracingStepSize(globalDccrgSize, stepSize); // In-flight storage of step size, needed when crossing into next MPI domain
+      
       std::array<int, 3> gridSize = technicalGrid.getGlobalSize();
-      uint64_t maxTracingSteps = 4 * (gridSize[0] * technicalGrid.DX + gridSize[1] * technicalGrid.DY + gridSize[2] * technicalGrid.DZ) / stepSize;
+      creal maxTracingDistance = 4 * (gridSize[0] * technicalGrid.DX + gridSize[1] * technicalGrid.DY + gridSize[2] * technicalGrid.DZ);
```

Comment:
Do we keep this heuristic (it triggers e.g. in FHA at 1000 s when tracing all cells) or do we want yet another cfg parameter?

Reply:
Well it's shaken out of a sleeve like "how far do we sensibly go in a box, thinking of not-too-complex IMF+dipole".